client/my-sites/customer-home/main.jsx

```diff
@@ -156,4 +156,4 @@ const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
 
 const connectHome = connect( mapStateToProps, mapDispatchToProps, mergeProps );
 
-export default flowRight( connectHome, withTrackingTool( 'HotJar' ) )( Home );
+export default compose( connectHome, withTrackingTool( 'HotJar' ) )( Home );
```

Comment:
We usually nest the HOCs like `connectHome( withTrackingTool( 'HotJar' )( Home ) )` when removing the `flowRight` helper.